### PR TITLE
Ability to toggle reactions container on-off on auditorium…

### PR DESCRIPTION
**Pull request by IvanBuljovcic**

### CODE REVIEW

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 1]
```diff
@@ -603,6 +604,23 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
           </div>
         )}
 
+        {templateID && HAS_REACTIONS_TEMPLATES.includes(templateID) && (
+          <div className="toggle-room">
```
[the remark this answers is not in the document]
Comment (IvanBuljovcic) [reply src/pages/Admin/Venue/DetailsForm.tsx:608]: Refactored in Admin v2 (self-serve), this specific component will be removed with the updates in that

File: functions/venue.js [hunk 2]
```diff
@@ -443,6 +448,10 @@ exports.updateVenue = functions.https.onCall(async (data, context) => {
     updated.showRangers = data.showRangers;
   }
 
+  if (typeof data.showReactions === "boolean") {
+    updated.showReactions = data.showReactions;
+  }
```
Comment (thecadams) [inline functions/venue.js:453]: You could do:

```
if (data.showReactions) {
  updated.showReactions = isTruthy(data.showReactions);
}
```

**Review by thecadams — APPROVED** [review]
LGTM.

File: functions/venue.js (hunk 2 above)
Comment (IvanBuljovcic) [reply functions/venue.js:453]: This approach works only if `data.showReactions` is true, if it is false, the `updated` will not get the value

**Pull request merged**